pkg/block/fetcher_test.go

```diff
@@ -334,14 +338,17 @@ func TestLabelShardedMetaFilter_Filter_Basic(t *testing.T) {
 	}
 
 	synced := promauto.With(nil).NewGaugeVec(prometheus.GaugeOpts{}, []string{"state"})
-	f.Filter(input, synced, false)
-
+	err := f.Filter(ctx, input, synced, false)
```

Comment:
what about `testutil.Ok(f.Filter(ctx, input, synced, false)`? (:+1:

Reply:
yes, have made this change at other places as well.